docs/api/model.md

```diff
@@ -1,6 +1,6 @@
 <a name="model"></a>
 # Class Model
-[View code](https://github.com/sequelize/sequelize/blob/d10eee53d37abb67db47160f067ac3cdc7e1bf43/lib/model.js#L26)
```

Comment:
Pl0x remove api docs from the PR - Its easier to review without, and easier to merge since there is less risk of a conflict